data_remote.json

```diff
@@ -9,9 +9,16 @@
     {
         "name": "rugby",
         "filename": "rugby.nc",
-        "url": "http://ndownloader.figshare.com/files/16254359",
-        "checksum": "9eecd2c6317e45b0388dd97ae6326adecf94128b5a7d15a52c9fcfac0937e2a6",
-        "description": "The Six Nations Championship is a yearly rugby competition between Italy, Ireland, Scotland, England, France and Wales. Fifteen games are played each year, representing all combinations of the six teams.\n\nThis example uses and includes results from 2014 - 2017, comprising 60 total games. It models latent parameters for each team's attack and defense, as well as a parameter for home team advantage.\n\nSee https://docs.pymc.io/notebooks/rugby_analytics.html by Peader Coyle for more details and references."
+        "url": "https://figshare.com/ndownloader/files/44916469",
```

Comment:
Should the figshare URL format for the other datasets also be updated, perhaps in another PR?

Reply:
it should be http everywhere, it will probably break otherwise. ArviZ itself replaces it with https by default https://github.com/arviz-devs/arviz/blob/main/arviz/data/datasets.py#L126